app/src/main/java/org/oppia/android/app/mydownloads/MyDownloadsViewPagerAdapter.kt

```diff
@@ -15,10 +20,14 @@ class MyDownloadsViewPagerAdapter(fragment: Fragment) :
   override fun createFragment(position: Int): Fragment {
     return when (MyDownloadsTab.getTabForPosition(position)) {
       MyDownloadsTab.DOWNLOADS -> {
-        DownloadsTabFragment()
+        val downloadsFragment = DownloadsFragment()
+        val args = Bundle()
+        args.putInt(INTERNAL_PROFILE_ID_SAVED_KEY, internalProfileId)
+        downloadsFragment.arguments = args
+        downloadsFragment
```

Comment:
Instead of implementing like this, suggest introducing a `newInstance` function in `DownloadsFragment` and using it here. You can see `OngoingTopicListFragment` implementation for reference.

Reply:
done

Reply:
This can be a good option for good first issues at multiple places. Will file tomorrow.